### ISSUE TITLE
Entry's asset URL show as tag_pair

### ISSUE THREAD

**Issue by vitor-ao** [statamic/cms#1714]
## Bug Description
Asset URL only shows on the first entry (even when sorted differently). The subsequent URLs show as `tag_pair_45dcd3bfd0e6abf6cf47ffe9cfd6e6b0`, even if the entry doesn't have an asset attached to it.

## How to Reproduce
- Create a collection with at least one asset field included in the blueprint.
- Add an asset or just leave it empty.
- Add asset URL inside collection and render on browser to display the bug:
```
{{ collection from="name"  }}
{{ title }} - {{ assetname }}{{ url }}{{ /assetname }}<br>
{{ /collection }}
```

## Environment

**Statamic version:** 3.0.0.-beta-26

**PHP version:** 7.3.13

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by francescomugnai** [comment]
Same problem here (after the update).
I'm getting the same error on the taxonomies and bard field.

**Comment by tao** [comment]
This might be related, when using statamic:ssg it's producing a similar output on the antlers syntax to prevent parsing `@{{ toggleModeIcon }}` it returns `noparse_2b3a1f15a7bfc975964283df7e6c7c2b`

**Comment by vitor-ao** [comment]
An update: If I limit to 1 asset ( `{{ assetname limit="1"}}{{ url }}{{ /assetname }}` ) the problem disappears.

**Comment by tao** [comment]
I've done some experimenting with this, and `synopsis` is a simple list field type... the following snippet works:

```html
<div class="max-w-2xl mx-auto mb-32">
    {{ collection:articles limit="5" as="articles" }}
        {{ articles }}
            <div class="mb-16">
                <h2 class="mb-4">
                    <a href="{{ url }}" class="text-2xl hover:text-teal tracking-tight leading-tight font-bold">
                        {{ title | widont }}
                    </a>
                </h2>

                <ul class="pb-2 text-gray-600">
                  {{ synopsis }}
                    <li class="text-sm italic">- {{ value }}</li>
                  {{ /synopsis }}
                </ul>
            </div>
        {{ /articles }}
    {{ /collection:articles }}
</div>
```

And renders the correct output:

```
Shrill night

- This is the Synopsis
- Lorem ipsum dolor sit amet
- Morbi tincidunt augue interdum
```

While this version doesn't work:

```html
<div class="max-w-2xl mx-auto mb-32">
    {{ collection:articles limit="5" }}
          <div class="mb-16">
              <h2 class="mb-4">
                  <a href="{{ url }}" class="text-2xl hover:text-teal tracking-tight leading-tight font-bold">
                      {{ title | widont }}
                  </a>
              </h2>
              <ul class="pb-2 text-gray-600">
                {{ synopsis }}
                  <li class="text-sm italic">- {{ value }}</li>
                {{ /synopsis }}
              </ul>
          </div>
    {{ /collection:articles }}
</div>
```

And renders:
```
Snowy frog

tag_pair_6d8118bd1daeb69f627c57fcd5059a3a
```

**Comment by jasonvarga** [comment]
Where you are doing `{{ some_field }} {{ /some_field }}` - does the tag_pair thing only appear where `some_field` is not defined?

**Comment by tao** [comment]
@jasonvarga you can try it yourself on the homepage in this [demo site](https://github.com/tao/statamic-tag-pairs).

You should see this:
![Screenshot 2020-05-01 at 18 22 07](https://user-images.githubusercontent.com/1446331/80825626-ad9a7880-8bd8-11ea-866b-dd05ede22763.png)

**Comment by jasonvarga** [comment]
Yup, thanks. Fixed in c06a77109a2413e5b060214feb4241ec2812924e
